manifest.json

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Why is this necessary?

Reply:
I think this was made with polymer init and this might be vesitigial